src/main/scala/beam/agentsim/agents/ridehail/RideHailAgent.scala

```diff
@@ -228,6 +232,33 @@ class RideHailAgent(
           .copy(currentVehicle = Vector(vehicle.id), remainingShifts = shifts.get)
       }
   }
+  when(OfflineForCharging) {
+    case ev @ Event(ParkingInquiryResponse(stall, _), data) =>
+      log.debug("state(RideHailingAgent.OfflineForCharging.ParkingInquiryResponse): {}", ev)
+      vehicle.useParkingStall(stall)
```

Comment:
I still have to run tests, but can anybody verify that this would be enough to move the vehicle to this location? Or do I need to first create a routing request and move the vehicle. IIRC the manager will handle that - but it doesn't hurt to verify with those who might know ;)